### PR TITLE
docs: conditions

### PR DESCRIPTION
**Pull request by Meir017**
How can these be used? (no usage in the code)
- `IfCondition Equal(IfCondition condition)`
- `IfCondition NotEqual(IfCondition condition)`
- `IfConditionBuilder<T>.Equal(string)`

Related #325

### CODE REVIEW

File: src/Sharpliner/AzureDevOps/ConditionedExpressions/IfConditionBuilder.cs [hunk 1]
```diff
@@ -17,8 +17,10 @@ internal IfConditionBuilder(Conditioned? parent = null)
     }
 
     /// <summary>
-    /// Specify any condition
+    /// Use this to specify any custom condition (in case you miss some operator or expression).
     /// </summary>
+    /// <param name="condition">The condition to specify.</param>
+    /// <returns>The new condition.</returns>
     public IfCondition Condition(string condition)
         => Link(new IfCustomCondition(condition));
```
Comment (Meir017) [inline src/Sharpliner/AzureDevOps/ConditionedExpressions/IfConditionBuilder.cs:26]: How can these be used? (no example in the code)

**Review by premun — APPROVED** [review]
LGTM, thanks!

**Comment by Meir017** [comment]
> How can these be used? (no usage in the code)
> 
> * `IfCondition Equal(IfCondition condition)`
> * `IfCondition NotEqual(IfCondition condition)`
> * `IfConditionBuilder<T>.Equal(string)`
> 

@premun can these methods be removed? I'm not sure how they can be used, these conditions require 2 parameters

**Comment by premun** [comment]
Ah, good find, missed that. Opened https://github.com/sharpliner/sharpliner/pull/341

**Pull request merged**